[Bug] Decryption failure causing instance initiation failure in migration stage. Unable to invoke Cipher due to bad padding

After the nightly image update on appsmith-ee  v1.9.20.5-SNAPSHOT both the nightly ee test instances are down due to the encryption keys error. This happened while applying migrations after the update.

`appsmithith-ee-fat-nightly-https | backend stdout | Caused by: io.mongock.api.exception.MongockException: Error in method[Migration109TransferToDatasourceStorage.executeMigration] : Unable to invoke Cipher due to bad padding`

--- a/app/server/appsmith-server/src/main/java/com/appsmith/server/migrations/db/ce/Migration109TransferToDatasourceStorage.java
+++ b/app/server/appsmith-server/src/main/java/com/appsmith/server/migrations/db/ce/Migration109TransferToDatasourceStorage.java
@@ -29,7 +29,7 @@ import static org.springframework.data.mongodb.core.query.Query.query;
  * to a new collection. The datasource will cease to have configurations after this point
  */
 @Slf4j
-@ChangeUnit(order = "109", id = "migrate-configurations-to-data-storage", author = " ")
+@ChangeUnit(order = "109", id = "migrate-configurations-to-data-storage-v2", author = " ")
 // Although this migration is common to CE and BE, the ordering of this change-unit needs to be specifically after 107-ee,
 // as this migration has EE overrides for default environmentId, and for getting the environments on existing workspaces, 107-ee needs to run
 public class Migration109TransferToDatasourceStorage {

